pkg/docker/docker.go

```diff
@@ -90,19 +90,17 @@ func NewClient(ctx context.Context) (Client, error) {
 	return generalClient{cli}, nil
 }
 
-func (g generalClient) GPUEnabled(ctx context.Context) (bool, error) {
-	info, err := g.Info(ctx)
+func (c generalClient) GPUEnabled(ctx context.Context) (bool, error) {
+	info, err := c.Info(ctx)
 	if err != nil {
 		return false, errors.Wrap(err, "failed to get docker info")
 	}
 	logrus.WithField("info", info).Debug("docker info")
-	if nv, ok := info.Runtimes["nvidia"]; ok {
-		return nv.Path != "", nil
-	}
-	return false, nil
+	nv, _ := info.Runtimes["nvidia"]
```

Comment:
https://go.dev/play/p/hyPjSxPOb35
It seems that getting a value from nil map won't panic in Go 1.18, maybe we can directly ignore the second returned value?

Reply:
@gaocegege PTAL.
Should I add some tests here?

Reply:
LGTM. I think it is hard to add tests here.

Reply:
There's a tricky way: make these two lines a function and add unit tests for the function

Reply:
Then I prefer to keep the code readers happy